zio-http/src/main/scala/zhttp/http/HttpData.scala

```diff
@@ -77,6 +77,11 @@ object HttpData {
   def fromStream(stream: ZStream[Any, Throwable, String], charset: Charset = HTTP_CHARSET): HttpData =
     HttpData.BinaryStream(stream.map(str => Unpooled.copiedBuffer(str, charset)))
 
+  /**
+   * Helper to create HttpData from a CharSequence
+   */
+  def fromCharSequence(text: CharSequence, charset: Charset = HTTP_CHARSET): HttpData = Text(text, charset)
```

Comment:
```suggestion
  def fromCharSequence(text: CharSequence): HttpData = Text(text, charset)
```

I think `fromCharSequence` doesn't need `Charset`